db/migrate/20170312171020_initialize_default_contexts.rb

```diff
@@ -0,0 +1,19 @@
+class InitializeDefaultContexts < ActiveRecord::Migration[5.0]
+  class PublishingContext < ActiveRecord::Base; end
+
+  def up
+    PublishingContext.create!(name: "milk_quantity")
+    PublishingContext.create!(name: "milk_quality")
+    PublishingContext.create!(name: "movement")
+    PublishingContext.create!(name: "temperature")
+    PublishingContext.create!(name: "intake")
+    PublishingContext.create!(name: "birth")
+    PublishingContext.create!(name: "calf")
+    PublishingContext.create!(name: "noise")
+    PublishingContext.create!(name: "health")
```

Comment:
Yeah I thought about that, but the things will likely never change. So I don't see the necessary overhead for implementing a web view.

Another solution is to implement it as enums, but then there is no chance of getting new contexts without deployment.